lib/config/presets/internal/workarounds.ts

```diff
@@ -292,4 +293,14 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  ubuntuDockerVersioning: {
+    description: 'Use ubuntu versioning for `node` docker images.',
```

Comment:
```suggestion
    description: 'Use ubuntu versioning for `ubuntu` docker images.',
```

wonder why this is not happening for azure-pipelines
https://github.com/renovatebot/renovate/blob/4032f206876960f36acfdce28982ac9167a48aa0/lib/modules/manager/dockerfile/extract.ts#L217-L219

Reply:
ah, it's regex manager 🤦‍♂️  in #33574